### ISSUE TITLE
Advent priest is not going into Stasis on death/dying on holy warrior connection

### ISSUE THREAD

**Issue by LaksonVell** [long-war-2/lwotc#603]
He is using stasis on my units, he is also using holy warrior. Killing a unit that has the holy warrior does not kill the advent priest. Advent priest also does not go into stasis on a killing blow in any circumstances (Templar kill, Ranger sword kill, gun death)

**Comment by Clibanarius** [comment]
Uh, only killing the PRIEST will kill the Holy Warrior-buffed unit. Not the other way around. And the Priests' Sustain ability only has a chance to proc, I dunno the rate but it's assuredly not 100%.

**Comment by LaksonVell** [comment]
You are right on the holy warrior part, that's my bad then. Haven't had a chance to test it the other way. But the stasis should be at 20% for basic priests, it's definitely not for me. I've tried killing priests in every way I thought, I haven't had a single on death stasis since starting this playthrough. Thanks for your help tho!

**Comment by Clibanarius** [comment]
Aye, I think 25% or so is what the Sustain should be. Maybe it's only an on-spawn potential to get a Sustain charge and you're toying with a Priest who doesn't have it? Try using Yet Another F1 mod to see maybe. I know I've seen plenty of it in my games so far though, so Sustain's in there and working (or at least was!), just it's a rarity that it'll proc.

**Comment by SirMrMister** [comment]
Something to consider to reduce RNG is to make Sustain always trigger (or at a much higher proc rate) but only if the priest's Stasis is not on cooldown.

**Comment by LaksonVell** [comment]
I have just managed to trigger it on a advanced priest. It could be I had ridiculous RNG as I have killed a dozen basic priests so far. Guess this issue can be closed, or turned into enhancement.

**Comment by copyrite** [comment]
Closing as Priest Sustain has been working fine in recent memory and was reworked to be based on overkill damage probably after the issue was opened.
